addon/-private/attr.ts

```diff
@@ -1,12 +1,13 @@
 import { computed } from '@ember/object';
 import { assert } from '@ember/debug';
 import { DEBUG } from '@glimmer/env';
-import recordDataFor from './system/record-data-for';
+import Model from './system/model/model';
+import { getRecordDataFor } from './system/model/record-data-map';
 /**
   @module ember-data
 */
 
-function getDefaultValue(record, options, key) {
+function getDefaultValue(options) {
```

Comment:
I thought I could drop `record` and `key` from these `arguments` but I realize now we use them when we call `options.defaultValue.apply(null, arguments)`. This fails no test though so gonna need to add one.

Reply:
It actually did fail a test, I just didn't notice. :(